packages/spor-react/src/input/NumericStepper.tsx

```diff
@@ -35,6 +35,8 @@ type NumericStepperProps = {
   stepSize?: number;
   /** Whether to show the number input when value is zero  */
   showZero?: boolean;
+  /** Name added to the aria-label of subtract and add buttons. */
+  ariaLabelContext?: { singular: string; plural: string };
```

Comment:
Når man setter konteksten, så antar jeg man skriver det inn manuelt? Vil det utgjøre en stor forskjell om vi forholder oss til en normal string kontra et objekt som kan inneholde singular og plural?

Reply:
Valgte å gå for denne løsningen for å få best mulig labels. På teksten trenger man som regel både entall og flertallsformen av konteksten